core/spec/models/spree/payment_spec.rb

```diff
@@ -387,6 +403,14 @@
           expect { payment.purchase! }.to raise_error(Spree::Core::GatewayError)
           expect(payment.capture_events.count).to eq(0)
         end
+
+        it "should not log response params" do
```

Comment:
would it be possible to write a shared example with this code? Basically, the whole test except for one single line that differs (`payment.purchase`, `#capture!` and `#authorize!`) is repeated 3 times.

Reply:
Done 👍